source/_posts/2016-09-14-what_the_tag_series_ep1_document_object.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "What the tag? Episode 1 - The Document object"
+category: html
+comments: true
+author:
+  name: 'Federico Kauffman'
+  email: 'federico@wyeworks.com'
+  twitter_handle: 'fedekauffman'
+  github_handle: 'fedekau'
+  image: '/images/team/federico-kauffman.jpg'
+  description: "Developer working with Ruby and Ember, passionate about technology, robotics and crossfit (Yes, I\'am crazy)."
+---
+
+Twenty three years ago as I was learning to walk something more important was happening, something that has been part of the technology we use every day, something that pretty much brought us where we are (in terms of web development), that was HTML's birth!
+
+In this series we will dig into the modern HTML's DNA (or as the people at [WHATWG](https://whatwg.org/) call it, [The Living Standard](https://html.spec.whatwg.org/multipage/dom.html)), we will look for things that are not usual to see or people just don't know because anyone told them, and we will try to explore them.
+
+<!--more-->
+
+I've been working with HTML for 2 years, but I am pretty sure I am going to learn as much as many people with this adventure, so don't worry, if you just started coding and you feel that your sites still look like [the world's worst website ever](http://www.theworldsworstwebsiteever.com/) we will learn, and that's the important thing.
+
+# The Document object
+
+The DOM (Document object model) standard defines a [Document](https://dom.spec.whatwg.org/#interface-document) interface which has many attributes and functions that will let you interact and know more about the document you are dealing with.
+
+**Please right click this website, click inspect, go to the console tab and try the things we learn as you read, you won't be breaking anything!**
+
+## Resource metadata management
+
+`document.referrer`
+
+This attribute returns the URL of the document from which the user navigated from or the empty string if there is no such information (for example if you typed the URL directly) or if this information was blocked using the `noreferrer` link type.
+
+`document.cookie`
+
+Very common these days, if you are not using it in your websites probably someone is using it on you. It is used to store the HTTP cookies for the current document, this can be set by you or it can be set automatically by the browser. In some cases it will throw an error, see the [definition of the cookie](https://html.spec.whatwg.org/multipage/dom.html#dom-document-cookie) for more info.
```

Comment:
Because this post is aimed to developers, when I say "you" I mean the developer doing something like 

```js
document.cookie = "bla=bla"
```

Do you think I should include an example?